docs/usage/limitations.md

```diff
@@ -4,7 +4,7 @@ There's a number of issues in Iris that are fundamental limitations of Iris and
 
 1. [Iris doesn't fully work with all drivers, notably Apple's drivers on macOS.](drivers.md)
 2. [Iris doesn't work with certain mods or features of mods, because these mods do things that Iris cannot support.](incompatible-mods.md)
-3. [Some shader packs have bugs and shortcoings that are not the fault of Iris](../ShaderpackBugs.md). Some of these are limitations in the way that the shader pack is designed.
+3. [Some shader packs have bugs and shortcoimgs that are not the fault of Iris](../ShaderpackBugs.md). Some of these are limitations in the way that the shader pack is designed.
```

Comment:
```suggestion
3. [Some shader packs have bugs and shortcomings that are not the fault of Iris](../ShaderpackBugs.md). Some of these are limitations in the way that the shader pack is designed.
```
Your type fix contains a typo still?

Reply:
I can't believe i did that, should be fixed now